Projects/Features/Scene/CreateGroupScene/SelectGroupPhoto/SelectGroupPhotoView.swift

```diff
@@ -73,19 +73,21 @@ extension SelectGroupPhotoView {
   @MainActor
   private func photoInFrame(
     for keyword: GroupData.Keyword,
-    selectedImages: [UIImage]
+    selectedPhotosInfo: [PhotoInfo]
   ) -> some View {
-    let galleryIcon = selectedImages.isEmpty ? DesignSystem.Icons.galleryPlusBlack : DesignSystem.Icons.galleryWhite
+    let galleryIcon = selectedPhotosInfo.isEmpty ? DesignSystem.Icons.galleryPlusBlack : DesignSystem.Icons.galleryWhite
     
     return keyword.frame
       .resizable()
       .aspectRatio(contentMode: .fit)
       .foregroundStyle(keyword.foregroundColor)
       .background {
-        if !selectedImages.isEmpty {
-          Image(uiImage: selectedImages[0])
-            .resizable(resizingMode: .stretch)
-            .overlay(Color.black.opacity(0.3))
+        if !selectedPhotosInfo.isEmpty {
+          if let image = UIImage(data: selectedPhotosInfo[0].data) {
+            Image(uiImage: image)
+              .resizable(resizingMode: .stretch)
+              .overlay(Color.black.opacity(0.3))
+          }
```

Comment:
```suggestion
        if !selectedPhotosInfo.isEmpty,
          let image = UIImage(data: selectedPhotosInfo[0].data) {
            Image(uiImage: image)
              .resizable(resizingMode: .stretch)
              .overlay(Color.black.opacity(0.3))
            }
```
이런 표현 어떠세요?
굳이 한번 더 if 블럭을 탈 필요는 없어 보여서요 🙋🏻

Reply:
아니면 safe subscript 사용해도 좋을 것 같아요!

Reply:
굿
```swift
if let firstPhotoInfo = selectedPhotosInfo[safe: 0],
   let image = UIImage(data: firstPhotoInfo.data) {
    Image(uiImage: image)
        .resizable(resizingMode: .stretch)
        .overlay(Color.black.opacity(0.3))
}
```
요렇게만 깔끔할듯하네요.
Collection의 safe subscript 플젝에 구현되있었는지 기억안나지만 없다면 확장해놔도 되겠네요 👍

Reply:
safe subscript가 있었군요! TCACoordinator 내에 Collection을 확장하여 safe subscript가 구현되어 있네요ㅎㅎ 
의견 감사합니다